src/pamiq_core/threads/thread_types.py

```diff
@@ -0,0 +1,18 @@
+from enum import Enum, auto
+
+
+class ThreadTypes(Enum):
+    CONTROL = auto()
+    INFERENCE = auto()
+    TRAINING = auto()
+
+    @property
+    def name(self) -> str:
```

Comment:
docstringを追加。
```suggestion
    def name(self) -> str:
        """Returns the name of thread type."""
```

本来関数のdocstringは引数と戻り値の方を明示する必要がありますが、 `property`はその宣言そのものからgetter/setterの役割が自明になるので、簡易に1行でdocstringを書くだけで構いません。

関数とする場合（これは冗長なので、あまり推奨されませんが）

```py
def get_name(self) -> str:
    """Retrieves the name of thread type.
    
    Returns:
        The name of thread type.
    """
```